specs/0117-email-opt-in-unsubscribe.md

```diff
@@ -0,0 +1,109 @@
+
+This will allow us to reach out within legal compliance to the wider audience of registered users for things such as surveys, online events, etc. Future major platform enhancements will be able to build off this work, and starting this early will provide enough time to ensure interested and active contributors are opted-in ahead of time.
+
+Value for users / problems solved:
+- Users can select their preference for email communication
+- Engagement/reach expanded beyond logged-in active Pontoon users
+
+
+# Feature explanation
+
+## Email communication user settings
+
+The email opt-in consent option is available under an “Email communications” option in user preferences. “Contact email address” should be moved from Personal information to under this section and an option for non-transactional mass emails opt-ing should be added.
+
+These user email communication preferences are stored and referenced in database to determine if emails should be sent to the user.
+
+
+(Section: content)
+- Section title: `Email communications`
+- Input field: Contact email address
+- Subtitle of input: `If provided, this email address will be used for email communications and will appear under your Profile page instead of the email used for login.`
+- Check box: `Receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform.`
+- Subtitle to check box: `I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`
+
+## Standalone page for email consent shown after existing user login
+
+We need to make existing users aware of the new email communication preference.
+
+Upon login (after a user’s previous session expires) an email consent page should be displayed to existing users logging in for the first time after the consent feature is enabled. 
+
+They should be shown two buttons, one to opt in to emails or one declining to opt in. This should not be shown again if the user clicks either of the buttons, but should be shown again upon next login if the user closed their tab/browser.
+
+This page also should not be shown if the user has already opted in via their settings.
+
+(Section: content)
+- Header: `Update your email communication preferences`
+- Sub-header: `You can now choose your email communication preferences within Pontoon.`
+- Body: `Would you like to receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform?`
+- Consent: `By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`
+- Button: `Enable email updates`
+- Button: `No, thank you`
+- Body: `You can update your email communication preferences at any time in your settings.`
```

Comment:
Perhaps we should put these into a mockup? "Body", "Consent" and another "Body" don't really tell much.